### PR TITLE
New Introduction for OmniBOR spec, and other changes.

### PR DESCRIPTION
**Pull request by alilleybrinker**
This PR actually includes multiple changes, including a reorganization of the project folder structure, the addition of new material to the `README.md`, formatting and grammatical improvements to most of the files in the project, and finally the introduction of a new Introduction section for the specification. These changes are logically split into a sequence of commits, and I recommend auditing them in commit order to help understand the reasoning, which is spelled out in each commit message.

### CODE REVIEW

**Comment by alilleybrinker** [comment]
I've edited the intro to incorporate key points from Ed's draft intro (#30).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
+public and that a record of the contribution (including all personal
+information I submit with it, including my sign-off) is maintained
+indefinitely and may be redistributed consistent with this agreement or the
+open source license(s) involved.
+
+I represent that I am legally entitled to make the grants set forth in the
+documents above.  If my employer(s) has rights to intellectual property that
+may be infringed by the materials developed by this Working Group, I
+represent that I have received permission to enter these agreements on behalf
+of that employer.
```
Comment (edwarnicke) [inline CONTRIBUTING.md:31]: This looks like basic reformatting to me, and all good :)

File: spec/SPEC.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+reliably capture a complete, concise, verifiable accounting of exactly
+__what__ inputs were built into software. Without this information,
+identifying vulnerable software to patch or replace is difficult. While
+Software Bills of Material (SBOMs) help identify third-party components,
+they do not go far enough to precisely identify the exact inputs necessary
+for vulnerability management.
+
+The OmniBOR standard defines three concepts, which together enable the
+consistent, reproducible, and embeddable encoding of the exact inputs used to
+build a software artifact: Artifact Identifiers, Input Manifests, and Artifact
```
Comment (edwarnicke) [inline spec/SPEC.md:57]: I would still favor 'Artifact Input Manifests' rather than 'Input Manifest'.

Comment (AevaOnline) [reply spec/SPEC.md:57]: There is a symmetry to each of the Three Concepts beginning with the word "Artifact" which I find appealing.

File: spec/SPEC.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+reproducible, meaning two individuals will always derive the same identifier
+for the same input. With these identifiers, we can consistently and precisely
+identify any software artifact or its input, for use in forensics, accounting,
+and vulnerability management.
+
+Next, an Input Manifest lists the Artifact Identifier of every input used to
+produce an artifact. For example, if an executable is compiled by linking
+together a collection of object files, the Artifact Identifier of every object
+file would be listed in the Input Manifest for the executable. Input manifests
+can be embedded directly into executable files, or can be provided in a
```
Comment (AevaOnline) [inline spec/SPEC.md:71]: > Input manifests can be embedded directly

I think what we want to say is that Input manifest _references_ can be embedded directly, not the entire manifest itself.

**Comment by AevaOnline** [comment]
Thank you, Andrew. I commented on two items inline, but otherwise LGTM!

**Pull request merged**